litellm/README.md

```
@@ -45,15 +107,24 @@ export PYTHONPATH="$LITELLM_DIR:$PYTHONPATH"
-Health: `curl http://127.0.0.1:4000/health/readiness` (unauth, fine). Models:
-`curl -H "Authorization: Bearer $LITELLM_MASTER_KEY" http://127.0.0.1:4000/v1/models`.
-Auth failure modes are ugly but closed (verified 2026-08-12): no key → 500 `internal_server_error`,
-wrong key → 400 `No connected db.`; neither serves the catalog or reaches upstream.
+Restart it with `launchctl kickstart -k gui/$(id -u)/com.chronicle.litellm`.
 
-### Testing a config change without touching the live spine
+## Verify
 
-Never restart the live spine to try an edit. Start a second instance on another loopback port against the
-candidate config, point a temp spend log at `/tmp` so Lane C's real capture stays clean, verify, then kill it.
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)